pkg/controller/admissionchecks/multikueue/multikueuecluster.go

```diff
@@ -157,6 +214,7 @@ var _ reconcile.Reconciler = (*clustersReconciler)(nil)
 
 func (c *clustersReconciler) Start(ctx context.Context) error {
 	c.rootContext = ctx
+	go c.runGC(ctx)
```

Comment:
For future work, I guess that it is possible to register runGC to the manager as a separate runnable.